examples/onnx/mplonnx2singa.py

```diff
@@ -0,0 +1,69 @@
+#
```

Comment:
File name is mplonnx2singa.py. but I don't know what is mpl.

Did you mean MLP instead? so the file name should be mplonnx2singa.py.